src/ui/translations/en.ts

```diff
@@ -88,7 +88,7 @@ Valid: from %4 to %5</source>
     </message>
     <message>
         <source>Retry?</source>
-        <translation>Retry?</translation>
+        <translation type="vanished">Retry?</translation>
```

Comment:
Should we remove the [vanished](https://stackoverflow.com/a/31892150/258772) translations?